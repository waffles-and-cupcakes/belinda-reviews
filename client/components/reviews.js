import React from 'react';
import ReactDOM from 'react-dom';
import axios from 'axios';

class Reviews extends React.Component {
  constructor(props) {
    super(props);

    this.state = {
      total: 5,
      reviews: []
    }
  }

  componentDidMount() {
    axios.get('/rooms/1/reviews')
    .then((res) => {
      console.log('RESPONSE: ', res);
      this.setState({
        reviews: res.data
      });
    })
    .catch((err) => {
      console.log('error GET-ing /rooms/1/reviews')
    })
  }

  render() {
    return (
      <div> {JSON.stringify(this.state.reviews)}</div>
    );
  }
}

export default Reviews;