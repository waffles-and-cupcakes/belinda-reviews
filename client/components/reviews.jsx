import React from 'react';
import ReactDOM from 'react-dom';


class Reviews extends React.Component {

  render() {
    return (
      <section className="review-container"> 

        <div className="detail-container">
          <img className="user-photo" src="./pfp.jpg" />
          <div className="x-detail-container">
            <p> Kristen </p>
            <p> June 2017 </p>
          </div>
        </div>

        <div className="review-body"> 
          A beautiful space! We loved all the attention to 
          detail and local recommendations. Would love to se all
            their properties!
        </div>

      </section> 
    )
    
  }
}

export default Reviews;