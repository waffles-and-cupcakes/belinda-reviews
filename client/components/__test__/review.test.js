import React from 'react';
import {shallow} from 'enzyme';
import Review from '../review';
import renderer from 'react-test-renderer';
import reviews from '../../mockData';


test('Reviews renders', () => {
  const component = renderer.create(
    <Review key={0} review={reviews[0]}/>
  )
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});