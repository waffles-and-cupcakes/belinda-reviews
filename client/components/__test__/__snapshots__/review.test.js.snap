// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reviews renders 1`] = `
<li
  className="collection-item avatar collection-item"
>
  <img
    className="circle"
    src="https://robohash.org/rerumnequeexplicabo.png?size=50x50&set=set1"
  />
  <span
    className="title"
  >
    <b>
      Rolland
    </b>
  </span>
  <p>
    September 2016
  </p>
  <a
    className="secondary-content"
    href="#!"
  >
    <i
      className="material-icons icon-flag"
    >
      flag
    </i>
  </a>
  <div>
    <span
      className="comment"
    >
      Vivamus tortor. Duis mattis egestas metus. Aenean fermentum. Donec ut mauris eget massa tempor convallis. Nulla neque libero, convallis eget, eleifend luctus, ultricies eu, nibh.
    </span>
  </div>
</li>
`;
